### ISSUE TITLE
NSString Extension -> String Extension

### ISSUE THREAD

**Issue by lfarah** [matteocrippa/MCFontAwesome#3]
Swift is a language that is trying to get as far as possible from Objective-C.
I'd be better to change the extension from NSString to String()

**Comment by matteocrippa** [comment]
yep, the repo is unmaintained for some months as soon as I have some spare time I'll update it
